[2023-05-10T02:07:57.494+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_25-30 scheduled__2022-05-05T23:00:00+00:00 [queued]>
[2023-05-10T02:07:57.503+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_25-30 scheduled__2022-05-05T23:00:00+00:00 [queued]>
[2023-05-10T02:07:57.505+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-10T02:07:57.506+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-10T02:07:57.513+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-10T02:07:57.534+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): scrape_teams_25-30> on 2022-05-05 23:00:00+00:00
[2023-05-10T02:07:57.540+0000] {standard_task_runner.py:55} INFO - Started process 17155 to run task
[2023-05-10T02:07:57.544+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_to_csv', 'scrape_teams_25-30', 'scheduled__2022-05-05T23:00:00+00:00', '--job-id', '4000', '--raw', '--subdir', 'DAGS_FOLDER/scrape_and_push_dag.py', '--cfg-path', '/tmp/tmpvipfg45q']
[2023-05-10T02:07:57.548+0000] {standard_task_runner.py:83} INFO - Job 4000: Subtask scrape_teams_25-30
[2023-05-10T02:07:57.649+0000] {task_command.py:389} INFO - Running <TaskInstance: scrape_and_save_to_csv.scrape_teams_25-30 scheduled__2022-05-05T23:00:00+00:00 [running]> on host f41dbfa3d325
[2023-05-10T02:07:57.691+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-10T02:07:57.738+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=scrape_and_save_to_csv
AIRFLOW_CTX_TASK_ID=scrape_teams_25-30
AIRFLOW_CTX_EXECUTION_DATE=2022-05-05T23:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-05T23:00:00+00:00
[2023-05-10T02:07:59.250+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:07:59.254+0000] {helpers.py:98} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:08:01.012+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:01.014+0000] {helpers.py:98} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:08:03.276+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:03.279+0000] {helpers.py:98} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:08:06.538+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:06.542+0000] {helpers.py:98} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:08:10.235+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:10.237+0000] {helpers.py:98} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:08:14.006+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:14.010+0000] {helpers.py:98} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:08:17.689+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:17.693+0000] {helpers.py:98} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:08:21.326+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:21.327+0000] {helpers.py:98} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:08:25.561+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:25.565+0000] {helpers.py:98} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:08:29.826+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:29.829+0000] {helpers.py:98} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:08:33.453+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:33.458+0000] {helpers.py:98} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:08:37.798+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:37.801+0000] {helpers.py:98} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:08:41.498+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:41.500+0000] {helpers.py:98} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:08:45.766+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:45.769+0000] {helpers.py:98} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:08:49.431+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:49.432+0000] {helpers.py:98} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:08:53.860+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:53.865+0000] {helpers.py:98} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:08:57.599+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:08:57.601+0000] {helpers.py:98} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:09:01.837+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:01.839+0000] {helpers.py:98} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:09:05.491+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:05.493+0000] {helpers.py:98} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:09:09.645+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:09.647+0000] {helpers.py:98} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:09:13.309+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:13.311+0000] {helpers.py:98} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:09:17.465+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:17.468+0000] {helpers.py:98} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:09:21.722+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:21.752+0000] {helpers.py:98} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:09:25.575+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:25.576+0000] {helpers.py:98} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:09:29.842+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:29.843+0000] {helpers.py:98} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:09:34.015+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:34.016+0000] {helpers.py:98} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:09:37.971+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:37.976+0000] {helpers.py:98} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:09:41.921+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:41.922+0000] {helpers.py:98} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:09:45.643+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:45.645+0000] {helpers.py:98} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:09:49.283+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:49.285+0000] {helpers.py:98} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:09:53.421+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:53.423+0000] {helpers.py:98} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:09:57.695+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:09:57.698+0000] {helpers.py:98} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:10:01.391+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:01.393+0000] {helpers.py:98} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:10:03.480+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:03.482+0000] {helpers.py:98} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:10:05.810+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:05.813+0000] {helpers.py:98} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:10:08.667+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:08.668+0000] {helpers.py:98} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:10:12.428+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:12.430+0000] {helpers.py:98} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:10:16.284+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:16.286+0000] {helpers.py:98} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:10:20.018+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:20.019+0000] {helpers.py:98} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:10:23.800+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:23.820+0000] {helpers.py:98} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:10:27.500+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:27.502+0000] {helpers.py:98} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:10:31.838+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:31.840+0000] {helpers.py:98} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:10:35.786+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:10:35.795+0000] {helpers.py:98} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:12:30.049+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:30.052+0000] {helpers.py:98} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:12:33.810+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:33.812+0000] {helpers.py:98} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:12:35.569+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:35.571+0000] {helpers.py:98} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:12:37.224+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:37.225+0000] {helpers.py:98} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:12:38.905+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:38.928+0000] {helpers.py:98} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:12:40.995+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:41.000+0000] {helpers.py:98} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:12:42.794+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:42.796+0000] {helpers.py:98} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:12:44.531+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:44.533+0000] {helpers.py:98} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:12:46.261+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:46.265+0000] {helpers.py:98} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:12:48.266+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:48.268+0000] {helpers.py:98} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:12:50.036+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:50.037+0000] {helpers.py:98} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:12:51.820+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:51.822+0000] {helpers.py:98} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:12:53.717+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:53.718+0000] {helpers.py:98} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:12:55.473+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:55.474+0000] {helpers.py:98} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:12:57.612+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:57.613+0000] {helpers.py:98} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:12:59.722+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:12:59.724+0000] {helpers.py:98} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:13:01.541+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:01.543+0000] {helpers.py:98} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:13:03.276+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:03.278+0000] {helpers.py:98} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:13:05.016+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:05.017+0000] {helpers.py:98} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:13:06.817+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:06.819+0000] {helpers.py:98} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:13:08.581+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:08.584+0000] {helpers.py:98} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:13:10.389+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:10.391+0000] {helpers.py:98} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:13:12.458+0000] {helpers.py:97} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:12.461+0000] {helpers.py:98} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:13:13.505+0000] {helpers.py:107} INFO - Saved in /opt/***/data/output_scrape_teams_25-30.csv
[2023-05-10T02:13:13.509+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-10T02:13:13.544+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=scrape_and_save_to_csv, task_id=scrape_teams_25-30, execution_date=20220505T230000, start_date=20230510T020757, end_date=20230510T021313
[2023-05-10T02:13:13.668+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-10T02:13:13.705+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
